src/style.css

```diff
@@ -0,0 +1,78 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+}
+
+.search {
+  position: relative;
+  margin-top: 20px;
+}
+
+.input {
+  width: 100%;
+  font-family: Avenir, sans-serif;
```

Comment:
move to `html` selector

Reply:
only font family?

Reply:
If I moved it to html, then for some reason they do not apply, even if I create a class for html , I don’t know why.
Therefore, I decided not to touch font-family.